reactions.json

```diff
@@ -29,7 +29,7 @@
     "AgF": "fluromethane",
     "CH3ONa": "methoxymethane"
   },
-  "bromomethane": {
+  "1-bromomethane": {

```

Comment:
bromomethane should be preferred over 1-bromomethane since it's redundant to write that 1